packages/components/src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,106 @@
+import type { SystemProp, Theme } from "@xstyled/styled-components";
+import React from "react";
+import * as HeroOutlineIcons from "@heroicons/react/24/outline";
+import { Box } from "../../primitives/Box";
+import { Icon } from "../Icon";
+
+type IconNames = keyof typeof HeroOutlineIcons;
+
+type BadgeSizeOptions = "large" | "small";
+type BadgeVariantOptions =
+  | "active"
+  | "inactive"
+  | "primary"
+  | "success"
+  | "warning";
+
+export interface BadgeProps
+  extends Omit<React.HTMLAttributes<HTMLDivElement>, "color"> {
+  /** The contents of the badge. Can be text. */
+  children: NonNullable<React.ReactNode>;
+  /** Sets the badge to be disabled. This is still being discussed with design whether we need the disabled prop or not. */
+  isDisabled?: boolean;
```

Comment:
I would prefer `disabled` because we are not starting boolean props with `is` in other components.

Reply:
Can a badge be disabled? It's just a div, right? It shouldn't be clickable, so no need to be disabled.

Reply:
They have the type disabled here https://www.figma.com/file/0KtV7fzjug9hL3XOLzqn1l/%5B-WIP-%5D-%F0%9F%AA%90-Pluto-Components?node-id=255%3A617 
Why can't a badge be disabled? If it's not clickable, it is disabled, right?

Reply:
Also I asked in the design channel and they said there is a disabled state: https://localyzeteam.slack.com/archives/C03DUMP3XD4/p1666624596425449